contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,69 @@
+[[contributing-user-stories]]
+= Contribute user stories to OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for OpenShift GitHub repository
+
+toc::[]
+
+== Modularization back story
+OpenShift docs are getting modularized, starting from OpenShift 4.0. Older content will
+be rewritten from scratch to conform to our modularization guidelines and new feature content
+will be created using user stories.
+
+== How do I contribute user stories?
+Familiarize yourself with the Customer Content Services (CCS)
+link:https://redhat-documentation.github.io/modular-docs/[modularization guide]. This is an important
+first step in making sure you are helping the docs team create content that is fully modularized.
+
+== What is the branch setup for modularized docs?
+
+* link:https://github.com/openshift/openshift-docs/tree/enterprise-4.0[enterprise-4.0]
++
+This builds both OCP 4.0 and Origin/Upstream content. Origin content will be separated using
+ifdef statements in content and distro statements in the _topic_map.yml.
++
+Only CCS submits and merges their PRs here till OCP 4.0 is released, unless others are keen to submit
+content in modularized format.
+
+* https://github.com/openshift/openshift-docs/tree/master[master]
+This builds legacy (existing), mostly non user story based content. This branch will continue to update
+docs.okd.io (upstream) till we switch to 4.0 with the enterprise-4.0 branch.
++
+Please continue to submit your PRs to this branch and specify if it is for 3.11 and/or 4.0.
+If the submitted PR is for the 3.11 branch, CCS merges with follow up edits in 3.11.
+CCS then creates user stories for its inclusion in enterprise-4.0 branch, and if it is completely new content,
```

Comment:
I thought we were only making stories for 4.0, and that looking at "3.11 stories" was just a mechanism to help us evaluate the sources of user stories. @aheslin?

If that's right, s/, and if it is completely new content,in the enterprise-3.11 branch as well./